nominal/core/checklist.py

```diff
@@ -170,6 +175,49 @@ def _from_conjure(cls, clients: _Clients, checklist: scout_checks_api.VersionedC
             _clients=clients,
         )
 
+    def execute_streaming(
+        self,
+        assets: Sequence[Asset | str],
+        notification_configurations: Sequence[str],
+        *,
+        evaluation_delay: timedelta = timedelta(),
+        recovery_delay: timedelta = timedelta(seconds=15),
+    ) -> None:
+        """Execute the checklist for the given assets.
+        - `assets`: Can be `Asset` instances, or Asset RIDs.
+        - `notification_configurations`: Integration RIDs, checklist violations will be sent to the specified
+           integrations.
```

Comment:
Maybe add here that we require there to be at least 1 notifactionConfiguration provided.